quickwit-config/resources/tests/server_config/server.toml

```diff
@@ -0,0 +1,13 @@
+metastore_uri = "postgres://username:password@host:port/db"
+
+[indexer]
+data_dir_path = "/opt/quickwit/data"
```

Comment:
we can set up here `IndexingSplitStoreParams` (there 2 params)

Reply:
Will do.